staging/src/k8s.io/kubectl/pkg/kuberc/kuberc.go

```diff
@@ -44,16 +41,8 @@ var (
 
 	aliasNameRegex = regexp.MustCompile("^[a-zA-Z]+$")
 	shortHandRegex = regexp.MustCompile("^-[a-zA-Z]+$")
-
-	scheme        = runtime.NewScheme()
-	strictCodecs  = serializer.NewCodecFactory(scheme, serializer.EnableStrict)
-	lenientCodecs = serializer.NewCodecFactory(scheme, serializer.DisableStrict)
 )
 
-func init() {
```

Comment:
Was this redundant?

Reply:
It wasn't, I moved it to `k8s.io/kubectl/pkg/config/scheme` where we register the scheme and types. This follows the pattern we have in other configs (see [kubelet](https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/apis/config/), [kube-controller-manager](https://github.com/kubernetes/kubernetes/tree/master/pkg/controller/apis/config/), [kube-proxy](https://github.com/kubernetes/kubernetes/tree/master/pkg/proxy/apis/config/), [kube-scheduler](https://github.com/kubernetes/kubernetes/tree/master/pkg/scheduler/apis/config/)).